tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -11,7 +11,8 @@ let deletedElements
 let timeOfLastDeleteOperation = Date.now()
 let timeOfLastUploadOperation = Date.now()
 const { download } = require('../helpers/webdavHelper')
-const { getFilesFoldersMatchingPattern, getFoldersMatchingPattern, getFilesMatchingPattern, getAllFilesStartingWithDot } = require('../helpers/filesFoldersHelper')
+const { getElementsMatchingPattern, getAllFilesStartingWithDot } = require('../helpers/filesFoldersHelper')
```

Comment:
Please modify this to be able to pass a filter, otherwise, there'll be an explosion of such functions (`getAllFilesNotStartingWithDot`, `getAllFilesContainingDots`, containing comma, etc, you get the idea).

Reply:
Looks like this was already here. Anyway, I think, it can be fixed here. :slightly_smiling_face:

Reply:
It has been used in previous prs as well. I will fix this in some specific seperate pr